altair/api.py

```diff
@@ -233,7 +233,10 @@ def configure_facet_scale(self, *args, **kwargs):
     def _ipython_display_(self):
         from IPython.display import display
         spec = self.to_dict()
-        data = {'application/vnd.vegalite+json': spec}
+        data = {
```

Comment:
Where is the sanitization being done? What if we use the VegaLite class here to do all of that?